example/client/include/KVMessageGenerator.hpp

```diff
@@ -0,0 +1,64 @@
+#pragma once
+
+#include <chrono>
+#include <map>
+#include <set>
+#include <vector>
+#include <cstdio>
+#include <cstring>
+#include <sys/param.h>
+#include <unistd.h>
+#include <yaml-cpp/yaml.h>
```

Comment:
please move the headers to .cpp if not required.

Reply:
fixed for all files.